Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN pdm sync -G dev
 
 FROM base as docs
 
-RUN pdm sync -G docs
+RUN pdm sync -G dev -G docs
```

Comment:
Without dev deps sphinx will not import `hiku.readers.graphql` because it can not import `graphql-core` library